docs/validators/guides/block-production.md

```diff
@@ -0,0 +1,84 @@
+# Block production
+Koinos uses the novel consensus algorithm, Proof-of-Burn (PoB). To participate in block production you will need to configure your node as well as perform some simple actions on the blockchain.
+
+---
+## Proof-of-Burn basics
+Proof-of-Burn has similarities to both Proof-of-Work (PoW) and Proof-of-Stake (PoS) with some notable differences. Like Proof-of-Stake, no mining occurs to produce blocks; this places all physical nodes on an equal playing field. Hashing is simulated in Proof-of-Burn by burning KOIN in order to gain Virtual Hash Power (VHP). Aside from using VHP rather than physical machines performing hashes, the algorithm works similarly to Proof-of-Work.
+
+---
+## Prerequisites
+Please follow our guide on [running a node](running-a-node.md). Be sure to start the node with the `--profile all` or `--profile block_producer` option to start the optional block producer microservice.
```

Comment:
Should we be recommending to set the profile in `.env` instead of the command line?